nemo/collections/asr/parts/utils/diarization_utils.py

```diff
@@ -154,7 +332,44 @@ def __init__(self, **cfg_diarizer):
             'white': '\033[0;37m',
         }
 
+        self.csv_columns = [
+            'uniq_id',
+            'DER',
+            'CER',
+            'FA',
+            'MISS',
+            'est_n_spk',
+            'is_spk_count_correct',
```

Comment:
instead of this isn't it better to directly have true speaker count value?

Reply:
changed